syntax = "proto3";
package yy;

message MsgToServer
{
    enum Mode{
        EXIT = 0;
        RUN = 1;
    }

    Mode msgMode = 1;
    int32 width = 2;
    int32 height = 3;
    
    repeated bytes imgBuffer = 4;
}

message MsgToClient
{
    int32 width = 1;
    int32 height = 2;
    string actionType = 3;
}